<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(13, 13, 13);
font: 24pt &quot;Verdana&quot;;</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Project_name">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 40pt &quot;PT Serif&quot;;
color: rgb(225, 226, 225);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;VK Analytics&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Group_id">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);
font: 13pt &quot;Verdana&quot;;
border-radius: 20%</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Instruction_1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(218, 219, 218);
font: 16pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>Введите &quot;id&quot; вашей группы:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Date_button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>230</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(210, 210, 210);
font: 16pt &quot;Verdana&quot;;
color: rgb(37, 34, 31);
border-radius: 10%
</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="information">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>640</y>
      <width>1461</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(62, 62, 62);
font: 13pt &quot;PT Serif&quot;;
color: rgb(201, 201, 201);
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#fcffff;&quot;&gt;Возможности сервиса:&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#c0c3c3;&quot;&gt;Предиктивный анализ лайков, комментариев и репостов&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#c0c3c3;&quot;&gt;Анализ роста/спада активности аудитории за определенный промежуток времени&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#c0c3c3;&quot;&gt;Наглядное представление аналитики в виде графиков&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;span style=&quot; text-decoration: underline; color:#ecf0f0;&quot;&gt;Подпишитесь на нашу группу ВКонтакте, чтобы быть в курсе последних новостей: https://vk.com/deadboy.timmy&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Instruction_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(218, 219, 218);
font: 16pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите дату от которой&lt;/p&gt;&lt;p&gt;надо проанализировать посты:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Instruction_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(218, 219, 218);
font: 16pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите дату до которой&lt;/p&gt;&lt;p&gt;надо проанализировать посты:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Date_from">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);
font: 13pt &quot;Verdana&quot;;
border-radius: 20%</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Date_to">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);
font: 13pt &quot;Verdana&quot;;
border-radius: 20%</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="PlotWidget" name="Plot_reposts">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>451</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="Plot_comms">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>340</y>
      <width>451</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="Plot_likes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>451</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Id_button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>100</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(210, 210, 210);
font: 16pt &quot;Verdana&quot;;
color: rgb(37, 34, 31);
border-radius: 10%
</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="Return">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>921</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(218, 219, 218);
font: 16pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Loading">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>110</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p/&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
